pkg/extensions/extensions.go

```diff
@@ -56,11 +56,11 @@ func EnableExtensions(extension *ExtensionConfig, log log.Logger, rootDir string
 
 // SetupRoutes ...
 func SetupRoutes(extension *ExtensionConfig, router *mux.Router, storeController storage.StoreController,
-	log log.Logger) {
+	log log.Logger, enableCVE bool) {
```

Comment:
In future, lets make this is a ... ExtensionOption

Reply:
I don't believe we need this parameter. We should already have what we need inside extension

Reply:
Yes, I will remove that